nixos/modules/services/desktops/accountsservice.nix

```diff
@@ -11,6 +11,7 @@ with lib;
   options = {
 
     services.accounts-daemon = {
+      restartIfChanged = false;
```

Comment:
Looks like this breaks eval because it belongs in the implementation (config below), not the options.

Reply:
I've copy-pasted it wrong from my systems nixpkgs. Fixed.